src/__tests__/getFunctionNameFromPath.test.ts

```diff
@@ -0,0 +1,16 @@
+import { getFunctionNameFromPath } from "@/ipc/processors/response_processor";
+import { describe, expect, it } from "vitest";
+
+describe("getFunctionNameFromPath", () => {
+  it("returns directory name for index files", () => {
+    expect(getFunctionNameFromPath("/foo/bar/index.ts")).toBe("bar");
+  });
+
```

Comment:
I don't know if this is the right expectation - when i look at supabase edge function examples (e.g. https://github.com/supabase/supabase/tree/master/examples/edge-functions/supabase/functions) it's always function-name/index.ts - is there supabase docs that says edge functions aren't always exported from index.ts?

Reply:
let's stick with the official convention. we can do broader support later if we get users requesting this. thanks